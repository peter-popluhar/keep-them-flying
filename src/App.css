@import-normalize;
@import '~antd/dist/antd.css';

.App {
  padding: 0 0 1.5rem;
}

