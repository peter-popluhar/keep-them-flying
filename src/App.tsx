import { BrowserRouter, Routes, Route } from "react-router-dom";

import "./App.css";
import Results from "./components/result";
import LandingPage from "./components/landing-page";
import Form from "./components/form";
import Banner from "./components/banner";
import Footer from "./components/footer";

function App() {
  return (
    <div className="App">
      <BrowserRouter>
        <Routes>
          <Route
            path="/"
            element={
              <LandingPage>
                <Banner>
                  <Form />
                </Banner>
              </LandingPage>
            }
          />
          <Route path="results" element={<Results />} />
        </Routes>
      </BrowserRouter>
      <Footer />
    </div>
  );
}

export default App;
