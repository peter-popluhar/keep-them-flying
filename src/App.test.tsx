import { render } from "@testing-library/react";
import App from "./App";

test("renders Button", () => {
  const { container } = render(<App />);
  expect(container).toMatchSnapshot();
});
