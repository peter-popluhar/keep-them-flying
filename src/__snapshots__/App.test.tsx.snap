// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Button 1`] = `
<div>
  <div
    class="App"
  >
    <div
      class="banner"
    >
      <h1
        class="heading"
      >
        Keep them flying...
      </h1>
      <div
        class="formGrid"
      >
        <div
          class="ant-skeleton ant-skeleton-element ant-skeleton-active"
        >
          <span
            class="ant-skeleton-input"
            style="width: 100%;"
          />
        </div>
        <div
          class="ant-skeleton ant-skeleton-element ant-skeleton-active"
        >
          <span
            class="ant-skeleton-input"
            style="width: 100%;"
          />
        </div>
        <div
          class="ant-skeleton ant-skeleton-element ant-skeleton-active"
        >
          <span
            class="ant-skeleton-input"
            style="width: 100%;"
          />
        </div>
        <div
          class="ant-skeleton ant-skeleton-element ant-skeleton-active"
        >
          <span
            class="ant-skeleton-input"
            style="width: 100%;"
          />
        </div>
        <div
          class="ant-skeleton ant-skeleton-element ant-skeleton-active"
        >
          <span
            class="ant-skeleton-input"
            style="width: 100%;"
          />
        </div>
        <div
          class="ant-skeleton ant-skeleton-element ant-skeleton-active"
        >
          <span
            class="ant-skeleton-input"
            style="width: 100%;"
          />
        </div>
        <div
          class="ant-skeleton ant-skeleton-element ant-skeleton-active"
        >
          <span
            class="ant-skeleton-input"
            style="width: 100%;"
          />
        </div>
        <div
          class="ant-skeleton ant-skeleton-element ant-skeleton-active"
        >
          <span
            class="ant-skeleton-input"
            style="width: 100%;"
          />
        </div>
        <div
          class="ant-skeleton ant-skeleton-element ant-skeleton-active"
        >
          <span
            class="ant-skeleton-input"
            style="width: 100%;"
          />
        </div>
        <div
          class="ant-skeleton ant-skeleton-element ant-skeleton-active"
        >
          <span
            class="ant-skeleton-input"
            style="width: 100%;"
          />
        </div>
      </div>
    </div>
    <div
      class="features"
    >
      <h2>
        Solutions for travelers during the COVID-19 pandemic
      </h2>
      <ul
        class="list"
      >
        <li>
          <img
            alt="Refunds & cancellations"
            src="https://www.kiwi.com/images/hero/icon-refunds.svg"
          />
          <h3>
            Refunds & cancellations
          </h3>
          <p>
            Learn more about our numerous refund options like carrier self-service and instant refunds.
          </p>
          <button
            class="ant-btn ant-btn-default"
            type="button"
          >
            <span>
              Find out more
            </span>
          </button>
        </li>
        <li>
          <img
            alt="Request a refund"
            src="https://www.kiwi.com/images/hero/icon-self-service.svg"
          />
          <h3>
            Request a refund
          </h3>
          <p>
            The fastest and easiest way to cancel your booking is online via our refunds & cancellations form.
          </p>
          <button
            class="ant-btn ant-btn-default"
            type="button"
          >
            <span>
              Manage my booking
            </span>
          </button>
        </li>
        <li>
          <img
            alt="Coronavirus information"
            src="https://www.kiwi.com/images/hero/icon-information.svg"
          />
          <h3>
            Coronavirus information
          </h3>
          <p>
            Read about how the new travel restrictions might affect your trip.
          </p>
          <button
            class="ant-btn ant-btn-default"
            type="button"
          >
            <span>
              COVID-19 information
            </span>
          </button>
        </li>
      </ul>
    </div>
    <div
      class="tiles"
    >
      <h2>
        Find inspiration in places picked just for you
      </h2>
      <ul
        class="list"
      >
        <li>
          <img
            alt="Barcelona"
            src="https://images.skypicker.com/?image=https%3A%2F%2Fimages.kiwi.com%2Fphotos%2F1280x720%2Fbarcelona_es.jpg&width=320&height=238&fit=cover&quality=75"
          />
          <h3>
            Barcelona
          </h3>
        </li>
        <li>
          <img
            alt="Athens"
            src="https://images.skypicker.com/?image=https%3A%2F%2Fimages.kiwi.com%2Fphotos%2F1280x720%2Fathens_gr.jpg&width=320&height=238&fit=cover&quality=75"
          />
          <h3>
            Athens
          </h3>
        </li>
        <li>
          <img
            alt="Palmas"
            src="https://images.skypicker.com/?image=https%3A%2F%2Fimages.kiwi.com%2Fphotos%2F1280x720%2Fpalma_es.jpg&width=320&height=238&fit=cover&quality=75"
          />
          <h3>
            Palmas
          </h3>
        </li>
        <li>
          <img
            alt="Berlin"
            src="https://images.skypicker.com/?image=https%3A%2F%2Fimages.kiwi.com%2Fphotos%2F1280x720%2Fberlin_de.jpg&width=320&height=238&fit=cover&quality=75"
          />
          <h3>
            Berlin
          </h3>
        </li>
      </ul>
    </div>
  </div>
</div>
`;
